build-farm/platform-specific-configurations/linux.sh

```diff
@@ -68,6 +68,9 @@ then
     then
       export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --enable-cuda --with-cuda=$CUDA_HOME"
     fi
+  elif [ "${ARCHITECTURE}" == "aarch64" ] && [ -f /usr/local/cuda-10.2/include/cuda.h ]; then
```

Comment:
maybe extract / pass in version?

Reply:
I'm not sure. I think it might be better to put it in the section with x64/ppc64le and have a switch for 9.0/10.2 in there depending on arch. Bear in mind these scripts are mostly to set the correct ones to ensure consistent builds on our platforms and not intended for end users who can specify whatever they want to `makejdk-any-platform.sh` so having it fixed isn't such a bad thing here.

Reply:
> I didn't look, but I would hope there's a way to arrange to use a different version, i.e. the hard-coded path doesn't override an explicitly specified `--with-code=/some/other/location`. Other than that, I have no objections.

There isn't, but that's no different from the other architectures :-)

Reply:
Yeah that's what I was thinking with my [earlier comment](https://github.com/adoptium/temurin-build/pull/2855#discussion_r1132529085) too. Yeah that seems reasonable - I wasn't certain whether it would override or not so I guess we're good with that :-)

Reply:
`aarch64` needs to be included in the test a few lines up as well.
https://github.com/adoptium/temurin-build/blob/master/build-farm/platform-specific-configurations/linux.sh#L56

Reply:
This is going to need a bit of a rebase after https://github.com/adoptium/temurin-build/pull/3350
@keithc-ca I'm not sure there's much point in having the architecture check you refer to in there at all to be honest. Worst case (And the only Linux it'll affect is s390x) it won't detect a cuda directory so won't build in the support.

Reply:
OK I've rebased and force-pushed an update which removes that check. Also removes the specific `CUDA_VERSION` and just sets `CUDA_HOME` directly in the two cases. I feel that keeps it simpler but if people disagree LMK.